vipps-recurring-api.md

```diff
@@ -19,7 +19,7 @@ If it is a critical issue, or involves sensitive information please
 |:-----|:----------------------------------------------- |
 | Agreement         | A payment subscription with a set of parameters that a customer agrees to  |
 | Charge         | A single payment within an agreement |
-| Idempotency | The property of endpoints to be called multiple times without changing the result after the initial request. |
+| Idempotency | The property of endpoints to be called multiple times without changing the result beyond the initial application. |
```

Comment:
Is there a reason for changing this?

Reply:
That is supposed to be the way it used to be. I'll revert it